requirements.txt

```diff
@@ -1,5 +1,5 @@
-pip==18.0.0
 pyglet==1.3.2
 numpy==1.15.2
 NBT==1.5.0
-prompt_toolkit==2.0.4
\ No newline at end of file
+git+git://github.com/Amulet-Team/Amulet-NBT.git
```

Comment:
Cython is listed in the requirements of linked repository. Make sure cython is installed. Without it setup.py for Amulet-NBT might fail, as it failed in circleci tests.

Reply:
I'll see if I can package the module into a `.whl` and go that route and if not I'll add in a workaround to use cython if it's installed, attempt to compile the generated `.c` files if cython isn't installed, and then fallback to a pure python version is all else fails